nomad/structs/structs.go

```diff
@@ -992,6 +992,25 @@ func (tg *TaskGroup) GoString() string {
 	return fmt.Sprintf("*%#v", *tg)
 }
 
+// The ServiceCheck data model represents the consul health check that
```

Comment:
Is it worth creating a block of const strings that enumerate the possible check types.
```
const (
    ServiceCheckHTTP = "HTTP"
    ServiceCheckTTL = "TTL"
....
)
```